README.md

```diff
@@ -36,12 +36,35 @@ Final render:
       <h1>Hello world</h1>
     </div>
 
+## React Native
+
 To activate React Native support you must pass in a `native` plugin option like so:
 
     plugins: [
       ["@fullstory/babel-plugin-annotate-react", { native: true }]
     ]
 
+### React Native with FullStory 
+
+
+When using this library with [FullStory for Mobile Apps](https://www.fullstory.com/platform/mobile-apps/), we recommend setting `setFSTagName: true` to generate better privacy selectors by setting [`fsTagName`](https://developer.fullstory.com/mobile/react-native/auto-capture/set-tag-name/) rather than `dataElement` and `dataComponent`.
```

Comment:
Suggested copy:

When using this library with [FullStory for Mobile Apps](https://www.fullstory.com/platform/mobile-apps/), we recommend setting `setFSTagName: true` to generate better privacy selectors. This setting will automatically set [`fsTagName`](https://developer.fullstory.com/mobile/react-native/auto-capture/set-tag-name/) with the value of `dataElement` or `dataComponent`, which will truncate the privacy selector and avoid duplicate naming.

Example:
* Before `RCTSafeAreaView[data-source-file="App.tsx"][data-element="SafeAreaView"][data-component="App"]`
* After `App[data-source-file="App.tsx"]`

(I hope this example is accurate).

We should probably put this `setFSTagName` setting in its own subtitle. Not sure why we added `React Native with FullStory` as it's a given that this plugin is only used with FullStory. We can just rename this section and dedicate it to describing this new setting.